omnibus.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency 'mixlib-versioning'
   gem.add_dependency 'ohai',             '~> 8.0'
   gem.add_dependency 'ruby-progressbar', '~> 1.7'
-  gem.add_dependency 'uber-s3'
+  gem.add_dependency 'uber-s3',          '~> 0.2.4'
```

Comment:
Can we just restrict this to `~> 0.2`? That will be the equivalent of `>= 0.2 && < 1.0`